BEST_PRACTICES.md

```diff
@@ -253,6 +254,19 @@ Views should be layout using Apple's Auto Layout. No third-party frameworks are
 
 Old style layout is still an option for when Auto Layout is not available.
 
+## Optional Force Unwrapping
+
+When something that shouldn't return an optional, returns an optional you have two options:
+
+**A)** Pass the error to the caller (throw) . 
```

Comment:
You could add _when_ to do this: 

`Pass the error to the caller (throw) , _if the app can recover._`
`Crash, _when the app can't recover_`